src/components/TodoComponents/TodoList.js

```diff
@@ -1,2 +1,27 @@
-// your components will all go in this `component` directory.
-// feel free to change this component.js into TodoList.js
+import React from "react";
+import Todo from "./Todo";
+import TodoForm from "./TodoForm";
+
+export default class TodoList extends React.Component {
+  render() {
+    return (
+      <>
+        <TodoForm
```

Comment:
To do so, pass the `todoValue` from App.js file to the `TodoForm` component. Like this => 
```
TodoForm
 handleInputChange={this.props.handleInputChange}
 handleTodoSubmit={this.props.handleTodoSubmit}
 **val={this.props.todoValue}**
 />
```